app/views/storage.html

```diff
@@ -30,11 +30,17 @@
             <div class="col-md-12">
               <div class="section-header page-header-bleed-right page-header-bleed-left">
                 <div class="hidden-xs pull-right" ng-if="project && ('persistentvolumeclaims' | canI : 'create')">
-                  <a ng-href="project/{{project.metadata.name}}/create-pvc" class="btn btn-default">Create Storage</a>
+                  <a ng-if="!outOfClaims" ng-href="project/{{project.metadata.name}}/create-pvc" class="btn btn-default">Create Storage</a>
+                  <div ng-if="outOfClaims" class="has-error">
+                    <span class="help-block">Exceeded quotas prevent creating new storage</span>
```

Comment:
So I don't think it should be an error here. Being at quota doesn't mean there's a problem. We have an alert we show when at quota on the overview page you might look at with a "Don't show me again" link.